dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -60,6 +60,47 @@ def __add__(self, other):
                                  other.__calculations)
         return merged_profile
 
+    def diff(self, other_profile, options=None):
+        """
+        Finds the differences for CategoricalColumns.
+
+        :param other_profile: profile to find the difference with
+        :type other_profile: CategoricalColumn
+        :return: the CategoricalColumn differences
+        :rtype: dict
+        """
+        differences = super().diff(other_profile, options)
+
+        differences["categorical"] = \
+            utils.find_diff_of_strings_and_bools(self.is_match, other_profile.is_match)
+
+        differences["statistics"] = dict([
+                ('unique_count', utils.find_diff_of_numbers(len(self.categories),
+                                                            len(other_profile.categories))),
+                ('unique_ratio', utils.find_diff_of_numbers(self.unique_ratio, 
+                                                            other_profile.unique_ratio)),
+            ])
```

Comment:
Should these all also only be calculated below if the column is categorical?

Reply:
No, these ones arent as intensive and also follow suit with the profile